tools/generate_code_cache.js

```diff
@@ -85,15 +81,24 @@ const cacheInitializers = [];
 const cacheHashInitializers = [];
 let totalCacheSize = 0;
 
+function lexical(a, b) {
+  if (a < b) {
+    return -1;
+  }
+  if (a > b) {
+    return 1;
+  }
+  return 0;
+}
 
-for (const key of cachableBuiltins) {
+for (const key of cachableBuiltins.sort(lexical)) {
   const cachedData = getCodeCache(key);
-  if (!cachedData.length) {
+  if (!(cachedData instanceof Uint8Array)) {
     console.error(`Failed to generate code cache for '${key}'`);
     process.exit(1);
   }
 
-  const length = cachedData.length;
+  const length = cachedData.byteLength;
```

Comment:
This change should be unneeded right?

Reply:
@TimothyGu Yeah, just so that it's clearer since this length is only used to calculate a string displayed in `KB`/`MB` etc. human readable format.

Reply:
I'll switch this to `const size = cachedData.byteLength` - I think that makes this a bit more self-explanatory